tiatoolbox/utils/visualization.py

```diff
@@ -12,9 +12,19 @@
 from PIL import Image, ImageFilter, ImageOps
 from shapely.geometry import Polygon
 
-from tiatoolbox import logger
+from tiatoolbox import DuplicateFilter, logger
 from tiatoolbox.annotation.storage import Annotation, AnnotationStore
 
+GEOMTYPES = {
+    1: "Point",
```

Comment:
shouldn't it start from 0?
```suggestion
    0: "Point",
```

Reply:
No, in the wkb specification it starts from 1. See:
https://www.ibm.com/docs/en/db2-warehouse?topic=formats-well-known-binary-wkb-format